### DISCUSSION TITLE
Local Kubermatic Kubernetes Platform (KKP) CE Installation error

### DISCUSSION THREAD

**Discussion by sunny092318** [kubermatic/kubermatic#12616] [General]
I'm trying to install the Local Kubermatic Kubernetes Platform (KKP) CE Installation on single on-prem host. But getting the below error. Could you please advise to fix the error. Below are document following for installations.
https://docs.kubermatic.com/kubermatic/v2.23/installation/local-kkp-installation/

Error log:

[root@kubes0101 ~]# ./kubermatic-installer local kind
INFO[0000] Creating kind cluster ...
FATA[0161] failed to create 'kind' cluster: exit status 1
Creating cluster "kkp-cluster" ...
• Ensuring node image (kindest/node:v1.27.3) 🖼 ...
✓ Ensuring node image (kindest/node:v1.27.3) 🖼
• Preparing nodes 📦 📦 ...
✓ Preparing nodes 📦 📦
• Writing configuration 📜 ...
✓ Writing configuration 📜
• Starting control-plane 🕹️ ...
✓ Starting control-plane 🕹️
• Installing CNI 🔌 ...
✓ Installing CNI 🔌
• Installing StorageClass 💾 ...
✓ Installing StorageClass 💾
• Joining worker nodes 🚜 ...
✗ Joining worker nodes 🚜
Deleted nodes: ["kkp-cluster-control-plane" "kkp-cluster-worker"]
ERROR: failed to create cluster: failed to join node with kubeadm: command "docker exec --privileged kkp-cluster-worker kubeadm join --config /kind/kubeadm.conf --skip-phases=preflight --v=6" failed with error: exit status 1

Command Output: I0901 08:42:14.959238 202 join.go:412] [preflight] found NodeName empty; using OS hostname as NodeName
I0901 08:42:14.959311 202 joinconfiguration.go:76] loading configuration from "/kind/kubeadm.conf"
I0901 08:42:14.962405 202 controlplaneprepare.go:225] [download-certs] Skipping certs download
I0901 08:42:14.962434 202 join.go:529] [preflight] Discovering cluster-info
I0901 08:42:14.962452 202 token.go:80] [discovery] Created cluster-info discovery client, requesting info from "kkp-cluster-control-plane:6443"
I0901 08:42:14.977486 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/namespaces/kube-public/configmaps/cluster-info?timeout=10s 200 OK in 14 milliseconds
I0901 08:42:14.978577 202 token.go:223] [discovery] The cluster-info ConfigMap does not yet contain a JWS signature for token ID "abcdef", will try again
I0901 08:42:21.277283 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/namespaces/kube-public/configmaps/cluster-info?timeout=10s 200 OK in 3 milliseconds
I0901 08:42:21.277858 202 token.go:223] [discovery] The cluster-info ConfigMap does not yet contain a JWS signature for token ID "abcdef", will try again
I0901 08:42:26.590064 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/namespaces/kube-public/configmaps/cluster-info?timeout=10s 200 OK in 1 milliseconds
I0901 08:42:26.590907 202 token.go:105] [discovery] Cluster info signature and contents are valid and no TLS pinning was specified, will use API Server "kkp-cluster-control-plane:6443"
I0901 08:42:26.590952 202 discovery.go:52] [discovery] Using provided TLSBootstrapToken as authentication credentials for the join process
I0901 08:42:26.590970 202 join.go:543] [preflight] Fetching init configuration
I0901 08:42:26.590978 202 join.go:589] [preflight] Retrieving KubeConfig objects
[preflight] Reading configuration from the cluster...
[preflight] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -o yaml'
I0901 08:42:26.601246 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/namespaces/kube-system/configmaps/kubeadm-config?timeout=10s 200 OK in 9 milliseconds
I0901 08:42:26.603712 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/namespaces/kube-system/configmaps/kube-proxy?timeout=10s 200 OK in 1 milliseconds
I0901 08:42:26.605104 202 kubelet.go:74] attempting to download the KubeletConfiguration from ConfigMap "kubelet-config"
I0901 08:42:26.606698 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/namespaces/kube-system/configmaps/kubelet-config?timeout=10s 200 OK in 1 milliseconds
I0901 08:42:26.608429 202 interface.go:432] Looking for default routes with IPv4 addresses
I0901 08:42:26.608443 202 interface.go:437] Default route transits interface "eth0"
I0901 08:42:26.608510 202 interface.go:209] Interface eth0 is up
I0901 08:42:26.608539 202 interface.go:257] Interface "eth0" has 1 addresses :[172.18.0.2/16].
I0901 08:42:26.608548 202 interface.go:224] Checking addr 172.18.0.2/16.
I0901 08:42:26.608556 202 interface.go:231] IP found 172.18.0.2
I0901 08:42:26.608563 202 interface.go:263] Found valid IPv4 address 172.18.0.2 for interface "eth0".
I0901 08:42:26.608569 202 interface.go:443] Found active IP 172.18.0.2
I0901 08:42:26.615951 202 kubelet.go:121] [kubelet-start] writing bootstrap kubelet config file at /etc/kubernetes/bootstrap-kubelet.conf
I0901 08:42:26.616611 202 kubelet.go:136] [kubelet-start] writing CA certificate at /etc/kubernetes/pki/ca.crt
I0901 08:42:26.616932 202 loader.go:373] Config loaded from file: /etc/kubernetes/bootstrap-kubelet.conf
I0901 08:42:26.617169 202 kubelet.go:157] [kubelet-start] Checking for an existing Node in the cluster with name "kkp-cluster-worker" and status "Ready"
I0901 08:42:26.618707 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/nodes/kkp-cluster-worker?timeout=10s 404 Not Found in 1 milliseconds
I0901 08:42:26.618977 202 kubelet.go:172] [kubelet-start] Stopping the kubelet
[kubelet-start] Writing kubelet configuration to file "/var/lib/kubelet/config.yaml"
[kubelet-start] Writing kubelet environment file with flags to file "/var/lib/kubelet/kubeadm-flags.env"
[kubelet-start] Starting the kubelet
[kubelet-start] Waiting for the kubelet to perform the TLS Bootstrap...
I0901 08:42:28.732321 202 loader.go:373] Config loaded from file: /etc/kubernetes/kubelet.conf
I0901 08:42:28.733796 202 cert_rotation.go:137] Starting client certificate rotation controller
I0901 08:42:28.734482 202 loader.go:373] Config loaded from file: /etc/kubernetes/kubelet.conf
I0901 08:42:28.735300 202 kubelet.go:220] [kubelet-start] preserving the crisocket information for the node
I0901 08:42:28.735329 202 patchnode.go:31] [patchnode] Uploading the CRI Socket information "unix:///run/containerd/containerd.sock" to the Node API object "kkp-cluster-worker" as an annotation
I0901 08:42:29.248601 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/nodes/kkp-cluster-worker?timeout=10s 404 Not Found in 12 milliseconds
I0901 08:42:29.738610 202 round_trippers.go:553] GET https://kkp-cluster-control-plane:6443/api/v1/nodes/kkp-cluster-worker?timeout=10s 404 Not Found in 2 milliseconds

Any leads will be appreciated, Thanks in advance

**Comment by embik** [comment]
Duplicate of #12615. Please do not open a discussion right after opening an issue with the same content.

**Comment by sunny092318** [comment]
sure, will close this one
